packages/lesmis-server/src/auth/index.ts

```diff
@@ -3,4 +3,4 @@
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-export { verifyLogin } from './verifyLogin';
+export { attachSession } from './attachSession';
```

Comment:
Very nitpicky, so totally optional, but I guess I feel this package ought to be renamed to `session` instead of `auth`?

Reply:
Valid!